### DISCUSSION TITLE
Ability to scope tags to certain models

### DISCUSSION THREAD

**Discussion by decoupca** [netbox-community/netbox#12002] [Ideas]
I've found most tags in my deployment only apply to certain models, such as devices, sites, or VMs. When applying tags to an object, the list of suggestions is cluttered with every available tag, most of which don't apply to that object.

It would be convenient to have the option to scope tags to certain models, the same way we can scope custom fields to one or more models. Limiting a tag's scope would remain optional; tags could still be assigned global scope (i.e., could be applied to any object, as they are now). This would make tagging objects more ergonomic and less error-prone.

Would anyone else find this useful?

**Comment by jeremystretch** [comment]
I think what you're describing has been proposed in FR #11541. If so, please be sure to give it a :+1:.

**Reply by decoupca** [reply]
That's exactly it, thanks!
